staff/ana-sanjuan/app-template/about.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About</title>
+</head>
+<body>
+    <header>
+        <h1>About</h1>
+        <nav>
+            <a href="home.html">Home</a> |
+            <a href="news.html">News</a> |
+            <a href="about.html">About</a> |
+            <a href="contact.html">Contact</a>|
+            <a href="register.html">Register</a> |
+            <a href="login.html">Login</a> |
+            <a href="profile.html">Profile</a> |
+        </nav>
+    </header>
+    <hr>
```

Comment:
why HR here? if you just add because of presentational reasons, the remove it. presentation is not cared here in this challenge.

Reply:
Ok!